cmake/modules/Host.cmake

```diff
@@ -1,4 +1,10 @@
-set(TEST_SUITE_HOST_CC "cc" CACHE STRING "C compiler targetting the host")
+
+if(WIN32 AND NOT TEST_SUITE_HOST_CC)
```

Comment:
Just a note here; `WIN32` indicates whether a cross compiler targets windows, while we probably want to know what host we're running on. A more accurate condition might be to check `CMAKE_HOST_SYSTEM_NAME` if that's what you're intending here?

Anyway, I'm not a fan of hardcoded defaults, but I think this probably is ok (and it can be overridden by the user when necessary).

Reply:
I have update this in new revision.